index.html

```diff
@@ -2540,9 +2540,45 @@ <h2>
   </p>
 </section>
 
-<section id="index" class="appendix">
+<section id="substantive-changes-summary" class="appendix informative">
+  <h2>Summary of substantive changes</h2>
+  <section id="substantive-changes-summary-fpwd">
+    <h3>Changes since the <a href="https://www.w3.org/TR/2022/WD-compute-pressure-20221220/">First Public Working Draft (2022/12/20)</a></h3>
+    <ul>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/302">Remove the limiting of rate as it is replace by obfuscation (PR #302)</a></li>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/300">Move at-risk to experimental on PressureSource/thermals (PR #300)</a></li>
+      <li><a href="https://github.com/w3c/compute-pressure/pull/299">Make ownContributionEstimate experimental (PR #299)</a></li>
```

Comment:
This entry and the next are not visible in the non-editors version, so can be removed